charts/consul/values.yaml

```diff
@@ -564,6 +565,9 @@ global:
     # from the one used by the Consul Service Mesh.
     # Please refer to the [Kubernetes Auth Method documentation](https://developer.hashicorp.com/consul/docs/security/acl/auth-methods/kubernetes).
     #
+    # If `externalServers.enabled` is set to true, then both `global.federation.k8sAuthMethodHost` and 
+    # `externalServers.k8sAuthMethodHost` should be populated and set to the same value.
```

Comment:
@thisisnotashwin following our conversation, I adjusted the language a bit here and below - I ended up landing on similar verbiage for the pair of comments, but instead of focusing on "secondary DC" it simply states the property match requirement. `federation` has direct references to secondary DCs, so that language didn't seem necessary in this addition.